simulationdataschema/model_system_review.py

```diff
@@ -0,0 +1,864 @@
+            - `Fm-3m`.
+        """,
+    )
+
+    strukturbericht_designation = Quantity(
+        type=str,
+        description="""
+        Classification of the material according to the historically grown and similar crystal
+        structures ('strukturbericht'). Useful when using altogether with `space_group_symbol`.
+        Examples:
+            - `C1B`, `B3`, `C15b`,
+            - `L10`, `L60`,
+            - `L21`.
+
+        Extracted from the AFLOW encyclopedia of crystallographic prototypes.
+        """,
+    )
+
+    prototype_formula = Quantity(
+        type=str,
+        description="""
+        The formula of the prototypical material for this structure as extracted from the
+        AFLOW encyclopedia of crystallographic prototypes. It is a string with the chemical
+        symbols:
+            - https://aflowlib.org/prototype-encyclopedia/chemical_symbols.html
+        """,
+    )
+
+    prototype_aflow_id = Quantity(
+        type=str,
+        description="""
+        The identifier of this structure in the AFLOW encyclopedia of crystallographic prototypes:
+            http://www.aflowlib.org/prototype-encyclopedia/index.html
+        """,
+    )
+
+    atomic_cell_ref = Quantity(
+        type=AtomicCell,
+        description="""
+        Reference to the AtomicCell section that the symmetry refers to.
+        """,
+        a_eln=ELNAnnotation(component="ReferenceEditQuantity"),
+    )
+
+    def resolve_analyzed_atomic_cell(
+        self, symmetry_analyzer: SymmetryAnalyzer, cell_type: str, logger: BoundLogger
+    ) -> Union[AtomicCell, None]:
+        """
+        Resolves the `AtomicCell` section from the `SymmetryAnalyzer` object and the cell_type
+        (primitive or conventional).
+
+        Args:
+            symmetry_analyzer (SymmetryAnalyzer): The `SymmetryAnalyzer` object used to resolve.
+            cell_type (str): The type of cell to resolve, either 'primitive' or 'conventional'.
+
+        Returns:
+            Union[AtomicCell, None]: The resolved `AtomicCell` section or None if the cell_type
+            is not recognized.
+        """
+        if cell_type not in ["primitive", "conventional"]:
+            logger.error(
+                "Cell type not recognized, only 'primitive' and 'conventional' are allowed."
+            )
+            return None
+        wyckoff = getattr(symmetry_analyzer, f"get_wyckoff_letters_{cell_type}")()
+        equivalent_atoms = getattr(
+            symmetry_analyzer, f"get_equivalent_atoms_{cell_type}"
+        )()
+        system = getattr(symmetry_analyzer, f"get_{cell_type}_system")()
+
+        positions = system.get_scaled_positions()
+        cell = system.get_cell()
+        atomic_numbers = system.get_atomic_numbers()
+        labels = system.get_chemical_symbols()
+
+        atomic_cell = AtomicCell(type=cell_type)
+        atomic_cell.lattice_vectors = cell * ureg.angstrom
+        atomic_cell.positions = positions * ureg.angstrom
```

Comment:
Why this conversion? Shouldn't the parser stipulate the units?

Reply:
This is coming from SymmetryAnalyzer

Reply:
There's something wrong if the `SymmetryAnalyzer` requires a fixed set of units (I reckon its precision)... Please add a note that we investigate this.